spring-cloud-stream/src/test/java/org/springframework/cloud/stream/aggregation/AggregationTest.java

```diff
@@ -82,7 +76,8 @@ public void closeContext() {
 	@Test
 	public void aggregation() {
 		aggregatedApplicationContext = new AggregateApplicationBuilder(
-				MockBinderRegistryConfiguration.class, "--server.port=0", "--debug=true")
+				FooConfig.class, "--server.port=0", "--debug=true",
```

Comment:
We have a `.web(false)`. What is the point of out of use `--server.port=0`?
And I believe we don't need `--debug=true`. That's what we used to determine the reason of failure in this test.